### PR TITLE
Remove dilations for pooling in onnx export and other small fixes

### PR DESCRIPTION
**Pull request by dzhulgakov**
- Dilation flag got removed
- Small changes to export functions to make life easier (used in onnx-pytorch)

### CODE REVIEW

**Comment by dzhulgakov** [comment]
@zdevito @ezyang (I wish authors could assign reviewers)

File: torch/onnx/__init__.py [hunk 1]
```diff
@@ -75,6 +75,29 @@ def export(model, args, f, export_params=True, verbose=False, training=False):
...
+def _trace(func, args, return_outs=False):
+    # Special case for common case of passing a single Variable
+    if isinstance(args, torch.autograd.Variable):
+        args = (args, )
+
+    trace, torch_out = torch.jit.trace(func, args)
+    _optimize_trace(trace)
+    if return_outs:
+        return trace, torch_out
+    return trace
```
Comment (apaszke) [inline torch/onnx/__init__.py]: If this is used only in onnx-pytorch, can't this go into onnx-pytorch?

**Review by houseroad — APPROVED** [review]
LGTM, thanks :-)

File: torch/onnx/__init__.py (hunk 1 above)
Comment (dzhulgakov) [reply torch/onnx/__init__.py]: It seems like a useful function in general if I want to export a Function, not a Module

**Pull request merged**